client/src/components/WelcomeHeader/WelcomeHeader.css

```diff
@@ -0,0 +1,13 @@
+.welcome-header {
+  width: 1440px;
+  height: 380px;
```

Comment:
This would be a great place to use `vh` and `vw`. We always want the header to be `100vh`. We'll want it to have a `min-height`, which could be `380px`. Otherwise, we'll want it to be some percentage of the view height (maybe 25 or 30 - what ever you think looks best!).

Reply:
Could you clarify how the width with vw would be handled? Setting the width to 100vw look s like this, with the background image repeating I just thought it looked weird.
<img width="1704" alt="Screen Shot 2019-12-08 at 1 42 47 PM" src="https://user-images.githubusercontent.com/55251251/70394209-bc318a80-19c0-11ea-8367-7860f0e4cd55.png">